src/main/resources/META-INF/maven/archetype-metadata.xml

```diff
@@ -250,5 +250,17 @@
         <include>.gitignore</include>
       </includes>
     </fileSet>
+    <fileSet filtered="true" encoding="UTF-8">
```

Comment:
Wouldn't these fit better in https://github.com/adobe/aem-project-archetype/blob/d1086465b65d06144424a3499595c52ca333732a/src/main/resources/META-INF/maven/archetype-metadata.xml#L122-L161 ?

Reply:
Fixed and updated this to have the fileset includes under `ui.apps`

@vladbailescu